client/src/util/socials.jsx

```diff
@@ -0,0 +1,33 @@
+import FacebookIcon from '../assets/social/facebook-square-brands.svg';
+import InstagramIcon from '../assets/social/instagram-square-brands.svg';
+import TwitterIcon from '../assets/social/twitter-square-brands.svg';
+import SnapchatIcon from '../assets/social/snapchat-square-brands.svg';
+import YoutubeIcon from '../assets/social/youtube-square-brands.svg';
```

Comment:
@natapokie it looks like this PR is adding 9 SVGs but only importing 5 - can you remove any unused ones?

Reply:
Yep! I imported the svg files again in the storyboard version of the footer, i just removed them so it should be good now!